cockroachcloud/backups-page.md

```diff
@@ -122,9 +200,57 @@ Additionally, you can [back up and restore](../v20.1/backup-and-restore.html) yo
     If you are backing up the data to AWS or GCP, use the `specified` option for the `AUTH` parameter.
     {{site.data.alerts.end}}
 
-1. To [restore](../v20.1/restore.html) to your CockroachCloud cluster:
+1. To [restore](../stable/restore.html) to your CockroachCloud cluster:
 
     {% include copy-clipboard.html %}
     ~~~ sql
     > RESTORE DATABASE example_database FROM 'gs://bucket_name/path_to_backup?AUTH=specified';
     ~~~
+
+## Troubleshooting
+
+### Resolve a database naming conflict
+
+The databases you want to restore cannot have the same name as an existing database in the target cluster. Before you restore a database, verify that the database name is not already in use. To do this, connect to your cluster with the [CockroachDB SQL client](connect-to-your-cluster.html#use-the-cockroachdb-sql-client) and run the following:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SHOW DATABASES;
+~~~
+
+If the database's name is already in use, either [drop the existing database](../stable/drop-database.html):
+
+{% include copy-clipboard.html %}
+~~~ sql
+> DROP DATABASE example_database;
+~~~
+
+Or [change the existing database's name](../stable/rename-database.html):
+
+{% include copy-clipboard.html %}
+~~~ sql
+> ALTER DATABASE example_database RENAME TO archived_example_database;
+~~~
+
+### Resolve a table naming conflict
+
+The table you want to restore cannot have the same name as an existing table in the target database. Before you restore a table, verify that the table name is not already in use. To do this, connect to your cluster with the [CockroachDB SQL client](connect-to-your-cluster.html#use-the-cockroachdb-sql-client) and run the following:
```

Comment:
same as above: might slightly edit "connect to your cluster" with "connect to the target cluster" for specificity